.github/workflows/build-multi-arch.yml

```diff
@@ -1,4 +1,4 @@
-name: Build x64 and ARM64 images 
+name: Build x86 and ARM64 images
```

Comment:
Why do we build x86?

Reply:
I believe existing workflow is set to build for x86 mainly for consistency purpose. In this case, build on x86 is also done in unit-test workflow.